nodes/Postiz/Postiz.node.ts

```
@@ -41,7 +41,7 @@ export class Postiz implements INodeType {
 						name: 'Create Post',
 						value: 'createPost',
 						description: 'Schedule a post to Postiz',
-						action: 'Schedule a post to postiz',
+						action: 'Schedule a post to Postiz',
```

ESLint

[error] Change to sentence case [autofixable]

(n8n-nodes-base/node-param-operation-option-action-miscased)